stdlib/public/runtime/Casting.cpp

```diff
@@ -1436,10 +1436,31 @@ extern "C" const _ObjectiveCBridgeableWitnessTable BRIDGING_CONFORMANCE_SYM;
 /// Nominal type descriptor for Swift.String.
 extern "C" const StructDescriptor NOMINAL_TYPE_DESCR_SYM(SS);
 
+struct ObjCBridgeWitnessCacheEntry {
+  const Metadata *metadata;
+  const _ObjectiveCBridgeableWitnessTable *witness;
+};
+
+// String is so important that we cache it permanently, so we don't want to
+// pollute this temporary cache with the String entry
```

Comment:
Is this comment out of date?  I don't see where anything here is specific to String.

Reply:
It's referring to how I updated the String memoization to call the no-cache version of this so that it doesn't end up in both places. It could probably be moved to a better location.